content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/_cq_dialog/.content.xml

```diff
@@ -72,6 +72,39 @@
                                         jcr:primaryType="nt:unstructured"
                                         sling:resourceType="granite/ui/components/coral/foundation/container">
                                         <items jcr:primaryType="nt:unstructured">
+                                            <headingType
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="/libs/granite/ui/components/coral/foundation/form/select"
+                                                fieldDescription="The heading type each accordion's label is."
+                                                fieldLabel="Heading Type"
+                                                name="./headingType">
+                                                <items jcr:primaryType="nt:unstructured">
+                                                    <h1
```

Comment:
I included an `h1` option but I'm wondering if this should be removed for SEO reasons. Does anyone have any thoughts/opinions on this?

Reply:
@brandonmaynard - what SEO problems do you foresee? H1-H6 seems fine to me, unless there's good reason we shouldn't restrict it.

Reply:
@richardhand, I thought for SEO, there should only be one `h1` per page and this property sets all Accordion Item Titles to the same configured heading value

Reply:
Right, I was imagining each accordion item being wrapped by a sectioning element, in which case it would be fine, but since we don't do that I think we can/should omit H1 as you say. Would love an expert SEO opinion though.

Reply:
Ok, I removed `h1`. Thanks for your assistance @richardhand